earthpy/io.py

```diff
@@ -117,51 +136,55 @@ def get_data(self, key=None, replace=False, url=None):
         -------
         path_data : str
             The path to the downloaded data.
+
+        Examples
+        --------
+        Download a dataset using a key:
+
+            >>> earthpy.data.get_data('california-rim-fire') # doctest: +SKIP
+
+        Or, download a dataset using a figshare URL:
+
+            >>> url = 'https://ndownloader.figshare.com/files/12395030'
+            >>> earthpy.data.get_data(url=url) # doctest: +SKIP
+
         """
         if key is not None and url is not None:
             raise ValueError(
                 "The `url` and `key` parameters can not both be "
                 "set at the same time."
             )
         if key is None and url is None:
-            print("Available datasets: {}".format(list(DATA_URLS.keys())))
+            print(self.__repr__())
             return
 
         if key is not None:
             if key not in DATA_URLS:
-                raise ValueError(
-                    "Don't understand key "
+                raise KeyError(
+                    "Key " + key + " not found in earthpy.io.DATA_URLS. "
                     "{}\nChoose one of {}".format(key, DATA_URLS.keys())
```

Comment:
print this as well to render the break?

Reply:
Fixed with 6f279daaf765204e9249521c96b368511d80d718